src/components/common/Drawer.js

```diff
@@ -42,6 +42,18 @@ class Drawer extends Component {
   render() {
     const { user } = this.props;
     const { role } = this.props;
+    let iconRole;
+    let iconText;
+    if (role === 'Gather' || role === 'Recolectar') {
```

Comment:
We shouldnt be comparing to 'Gather' or 'Recolectar' explicit values. We should be comparing to generic strings (i.e string.Gather), adding the corresponding entries to the localization/es.js and localization/en.js files. Reason being that our code will look cleaner and if we'd like to support more languages it'll be easier to do so.

Same can be done for the rest of the role comparisons.